src/streamlink/plugins/canalplus.py

```diff
@@ -6,23 +6,36 @@
 
 
 class CanalPlus(Plugin):
-    API_URL = 'http://service.canal-plus.com/video/rest/getVideos/{0}/{1}?format=json'
-    CHANNEL_MAP = {'canalplus': 'cplus', 'c8': 'd8', 'cstar': 'd17', 'cnews': 'itele'}
+	# NOTE : no live url for the moment
+    API_URL = 'https://secure-service.canal-plus.com/video/rest/getVideosLiees/cplus/{0}?format=json'
```

Comment:
https://secure-service.canal-plus.com/video/rest/getVideos/cplus/1469050?format=json
still works, it's better than the list api